swiftly/client/standardclient.py

```diff
@@ -558,10 +560,7 @@ def request(self, method, path, contents, headers, decode_json=False,
                     value = resp.read()
                     resp.close()
             except Exception as err:
-                status = 0
-                reason = '%s %s' % (type(err), str(err))
-                hdrs = {}
-                value = None
```

Comment:
Doing this then raising a new exception later breaks the traceback; instead just re-raise for now to make the diff easier to read.